Eth-Crowdfund/eth_crowdfund_be
# Campaign contributors joins

## What's this PR do?
* Adds the contributor and campaign_contributors tables to the database
* Adds the endpoints for adding a contributor to a campaign and viewing campaigns by contributor address
## Where should the reviewer start? 
* `README.md`
* `src/models/Contributor.py`
* `src/models/CampaignContributor.py`
* `src/tests/test_campaign.py`
* `src/tests/test_contributor.py`
* `src/views/CampaignView.py`
* `src/views/ContributorView.py`
## How should this be manually tested?
* Reviewer can complete a database migration and run `pytest` to verify 19 passing tests
* Reviewer can fire up their server and hit the new endpoints to verify functionality
## Any background context you want to provide?
* FE requested joins functionality between contributors and campaigns
## What are the relevant tickets?
* Closes #46 
* Closes #42 
* Closes #52
